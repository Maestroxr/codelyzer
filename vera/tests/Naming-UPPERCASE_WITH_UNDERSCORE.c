
/*OK*/
struct first{
	int something;
}
/*********/


/*PROBLEM*/
struct second{
	int somethingElse = 2;
}
/*********/

int main() 
{
	return 0;
	
}


