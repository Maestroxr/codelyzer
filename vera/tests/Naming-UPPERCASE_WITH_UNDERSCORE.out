C:\Work\AutoAssessment\codelyzer\vera\tests\Naming-UPPERCASE_WITH_UNDERSCORE.c:3: Curly bracket tab indentation incorrect, found 12 characters but expected 0 tabs.
C:\Work\AutoAssessment\codelyzer\vera\tests\Naming-UPPERCASE_WITH_UNDERSCORE.c:5: Closing curly bracket not in the same line or column, or brackets are not both indented with tabs.
C:\Work\AutoAssessment\codelyzer\vera\tests\Naming-UPPERCASE_WITH_UNDERSCORE.c:10: Curly bracket tab indentation incorrect, found 13 characters but expected 0 tabs.
C:\Work\AutoAssessment\codelyzer\vera\tests\Naming-UPPERCASE_WITH_UNDERSCORE.c:12: Closing curly bracket not in the same line or column, or brackets are not both indented with tabs.
